tests/vmi_hook_sidecar_test.go

```diff
@@ -52,9 +53,15 @@ import (
 
 const (
 	hookSidecarImage     = "example-hook-sidecar"
+	sidecarShimImage     = "sidecar-shim"
 	sidecarContainerName = "hook-sidecar-0"
+	configMapName        = "my-config-map"
+	configMapKey         = "my_script"
 )
 
+//go:embed testdata/sidecar-hook-configmap.sh
+var configMapData string
```

Comment:
Can we just use multi-line string?

Reply:
It was suggested in https://github.com/kubevirt/kubevirt/pull/10637#pullrequestreview-1701939805 to use embed. Are you suggesting this because we're creating `testdata` directory for a single file? Or, can you help understand the reasoning behind your suggestion?

Reply:
Yes, I don't see the need to use `embed` .

Reply:
Well, we can disagree. No problem.

IMHO, having code in its own file is helpful to copy/paste see it (highlight) and verify. It isn't a matter of number of lines.

Reply:
As one more opinion: I am more in favor of using `embed` in such cases. Mixing different programming languages in one file makes it harder to read the code (maybe not exactly with this script, but in general case). By opening the embedded data in a decoupled file it is easy to setup e.g. syntax highlighting and if the script gets more complicated, it is less error-prone to maintain it in a separate file.

Reply:
@xpivarc can you PTAL and suggest?

Reply:
@dharmit up to you.

Reply:
I prefer `go:embed` over a multi-line script. Can you please bless the PR with an LGTM if it's otherwise good? 😉